### DISCUSSION TITLE
API docs now live at recastnav.com

### DISCUSSION THREAD

**Discussion by grahamboree** [recastnavigation/recastnavigation#609] [Announcements]
Hi everyone!  Since the docs are frequently referenced and require some annoying setup steps to view locally, I added a GitHub action that automatically builds and deploys them from the main branch to [recastnav.com](http://www.recastnav.com) for easy reference.

The hope is that we'll be able to build out this domain as a more welcoming home for the project with additional info like tutorials, updates, releases, project showcases, etc.  For now, this should hopefully make our existing doxygen docs massively more useful than they have been.

**Comment by jankrassnigg** [comment]
The url isn't valid (anymore?)

**Comment by MaxHayman** [comment]
Works for me.

**Comment by grahamboree** [comment]
Hmm maybe there was a DNS hitch or something?  Seems to be working for me.

**Comment by jankrassnigg** [comment]
Indeed, works now for me as well. Weird.
